source/sonata_simulation.rst

```diff
@@ -513,7 +515,7 @@ List of dictionaries to adjust the synaptic strength or other properties of edge
    target                         text       Mandatory    node_set specifying postsynaptic nodes.
    weight                         float      Optional     Multiplier of ``conductance`` used to adjust synaptic strength. The final [NetCon weight](https://nrn.readthedocs.io/en/8.2.6/python/modelspec/programmatic/network/netcon.html#NetCon.weight) in NEURON is computed as ``weight * conductance``, where ``conductance`` is obtained from the ``edges.h5`` file.
    spont_minis                    float      Optional     Synapses affected by this connection_override section will spontaneously trigger with the given rate.
-   synapse_configure              text       Optional     Provide a snippet of hoc code which is to be executed on the synapse objects affected by this connection_override. Use '%s' to indicate where a reference to the synapse object should be filled.
+   synapse_configure              text       Optional     Provide a HOC snippet for synapse objects in ``connection_override``, using ``%s`` for the synapse reference. Global variables can be overridden without ``%s`` (e.g., ``tau_d_NMDA_ProbAMPANMDA_EMS = 2``), but we strongly suggest to limit overrides to **one per variable**. These changes **do not** account for delays, synapse type, or file order (which may depend on the edge file order). Use with caution, as unintended interactions may occur. Prefer modifying global variables via :ref:`modification_params`. However, this method takes priority.
```

Comment:
Sorry for the confusion. `Prefer modifying global variables via conditions:mechanisms` sections, not the `modification_params`. There is no separate section for `mechanisms`, perhaps you can reference to the `example::` part ?

Reply:
By the way, does **one per variable** mean that each "synapse_configure" shall only configure one variable ?

Reply:
good point. It is ambyguous. I will change it

Reply:
what do you thinnk about this table cell now?

Reply:
> Sorry for the confusion. `Prefer modifying global variables via conditions:mechanisms` sections, not the `modification_params`. There is no separate section for `mechanisms`, perhaps you can reference to the `example::` part ?

The text is clear now. But the reference is not corrected. See my comment above.

Reply:
yes, looks good!